spec/models/user_spec.rb

```diff
@@ -367,4 +366,26 @@
       it { expect(subject.admin?).to be_truthy }
     end
   end
+
+  describe '#full_name' do
+    let(:user) { build(:user, first_name: 'first', last_name: 'last') }
```

Comment:
I am using `build` here instead of `create` because it will not touch the database and run faster.

See: https://www.safaribooksonline.com/a/rails-4-test/9295406/

Reply:
👍